scripts/DR_distributions_cross_run_analysis.py

```diff
@@ -0,0 +1,228 @@
+         #'run160': 'v2.9',
+         #'run253': 'v2.10',
+         #'run161': 'v2.11',
+         #'run340': 'v2.12',
+         'run254': 'v2.13',
+         'run171': 'v2.14',
+         'run347': 'v2.15',
+         'run261': 'v2.16'}
+
+
+if __name__ == '__main__':
+
+
+    ###### step 1: get parcel_id - geographies mapping
+
+    p_geo = pd.read_csv('util\\2020_09_21_parcels_geography.csv',
+                        usecols = ['PARCEL_ID', 'juris',
+                                   #'fbpzoningm', 'fbp_gg_id', 'pda_id_pba50_fb', 'fbp_tra_id', 
+                                   'fbp_sesit_id'])
+    juris_county = pd.read_csv('util\\juris_county_id.csv', 
+                               usecols = ['juris_name_full', 'baus_output_juris_name', 'county_name'])
+
+    p_taz = pd.read_csv('util\\2020_08_17_parcel_to_taz1454sub.csv',
+                        usecols = ['PARCEL_ID', 'ZONE_ID'],
+                        dtype = {'PARCEL_ID': int, 'ZONE_ID': int})
+
+    taz_sd = pd.read_csv('util\\taz_geography.csv', usecols = ['zone','superdistrict'])
+    taz_sd.columns=['zone','sd_number']
+
+    sd = pd.read_csv('util\\superdistricts.csv', usecols = ['number', 'name', 'subregion'])
+    sd.columns = ['sd_number','superdistrict','subregion']
+
+    p_geo = p_geo.merge(juris_county,
+                        left_on='juris',
+                        right_on='juris_name_full',
+                        how='left').merge(p_taz,
+                                          on='PARCEL_ID',
+                                          how='left').merge(taz_sd,
+                                                            left_on='ZONE_ID',
+                                                            right_on='zone',
+                                                            how='left').merge(sd,
+                                                                              on='sd_number',
+                                                                              how='left')
+    print(p_geo.head())
+
+
+    ###### step 2: create index variables
+
+    juris = list(p_geo.juris.unique())
+    superdistrict = p_geo.superdistrict.unique()
+    zone = p_geo.zone.unique()
+    hra = ['HRA','non-HRA']
+    dis = ['DIS','non-DIS']
+    county = juris_county.county_name.unique()
+    year = list(range(2015, 2055, 5))
+    dr_type = ['deed_restricted_units', 'inclusionary_units', 'preserved_units', 'subsidized_units']
+
+
+    # set up tables for all DR cross-year comparison
+    # note: DR source breakdown table is set up within the loop in step 3
+
+    files = dict()
+
+    all_idx = {'dr_county_all': county,
+               'dr_juris_all': juris,
+               'dr_sd_all': superdistrict,
+               'dr_taz_all': zone,
+               'dr_hra_all': hra,
+               'dr_dis_all': dis}
+
+    for tablename in all_idx.keys():
+        table = pd.Series(np.nan, index = all_idx[tablename], name='temp')
+        files[tablename] = table
+
```

Comment:
Could we drop in a comment about what `files` is here and what the goal is.

e.g. files is a dictionary of table name -> Series of np.nans for each possible value.  
 so `dr_hra_all` -> Series('HRA': nan, 'non-HRA':nan))
The idea is we will replace the nan with a unit count of deed restricted units.